### ISSUE TITLE
[gradle] Gradle skips all tasks on --dry-run

### ISSUE THREAD

**Issue by manikmagar** [jreleaser/jreleaser#1061]
### Task List

- [x] Steps to reproduce provided
- [ ] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Use Gradle Plugin 1.3.1 - `id 'org.jreleaser' version '1.3.1'`
2. Execute a dry-run - `./gradlew jreleaserFullRelease --dry-run`

### Expected Behaviour

JReleaser to run all tasks with `dry-run set to true` mode. 

### Actual Behaviour

Gradle skips all tasks including the jreleaser tasks -

```
./gradlew jreleaserFullRelease --dry-run      

> Configure project :
[INFO]  Reading configuration
:compileJava SKIPPED
:processResources SKIPPED
:classes SKIPPED
:jar SKIPPED
:startScripts SKIPPED
:distTar SKIPPED
:distZip SKIPPED
:assembleDist SKIPPED
:jreleaserFullRelease SKIPPED
```

### Environment Information

- **Operating System**: Mac OS X 12.5 aarch64
- **JReleaser Version:** 1.3.1
- **JDK Version:** Java 11
```
./gradlew -v                                                                    

------------------------------------------------------------
Gradle 7.4
------------------------------------------------------------

Build time:   2022-02-08 09:58:38 UTC
Revision:     f0d9291c04b90b59445041eaa75b2ee744162586

Kotlin:       1.5.31
Groovy:       3.0.9
Ant:          Apache Ant(TM) version 1.10.11 compiled on July 10 2021
JVM:          11.0.16.1 (Azul Systems, Inc. 11.0.16.1+1-LTS)
OS:           Mac OS X 12.5 aarch64
```

Please paste stacktraces from `[out|build|target]/jreleaser/trace.log`.
No stacktraces available.

**Comment by manikmagar** [comment]
I see this old issue #800 but that seems to suggest same argument `--dry-run` that does isn't working.

**Comment by aalmiray** [comment]
Gradle has its own `--dry-run` flag which takes precedence over JReleaser's flag.

```
// from gradle --help
-m, --dry-run                      Run the builds with all task actions disabled.
```

This means JReleaser will have to revert to `--dryrun`.

**Comment by manikmagar** [comment]
Thank you @aalmiray for fixing this. Just wondering If there is a chance of this releasing earlier than v1.4.0 targetted for Dec-30?

**Comment by aalmiray** [comment]
There's an alternative that you may use: set the value of `jreleaser.dryRun` using a project property. This way you don't have to wait for the fix to be released, for example

```
def isDryRunEnabled = (project.hasProperty('dryRun')? project.ext.dryRun: false).toBoolean()

jreleaser {
  dryRun = isDryRunEnabled
  ...
}
```

Then you simply have to set the property, either in `gradle.properties` or in the command line

```
gradle -PdryRun=true jreleaserFullRelease
```

**Comment by aalmiray** [comment]
Released in v1.4.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.4.0
